pkg/client/config_test.go

```diff
@@ -179,25 +178,14 @@ func CreateTestConfigProvider(t *testing.T, cfg *config.Config) (config.Provider
 
 func TestFindClientConfigs(t *testing.T) {
 	t.Parallel()
```

Comment:
We cannot run this test in parallel with others because it replaces the global logger. If multiple tests do this, it's not clear which logger will be used.